test/specs/elements/Label/Label-test.js

```diff
@@ -45,17 +45,6 @@ describe('Label', () => {
   })
 
   describe('removeIcon', () => {
-    it('has no icon without onRemove', () => {
-      shallow(<Label />)
-        .should.not.have.descendants('Icon')
-    })
-
-    it('has delete icon by default', () => {
```

Comment:
> I would think then that the issue is that there should not be a default, since having one would break certain use cases. Let me know your thoughts on this.

As per this comment, I removed the default entirely so you'd have to pass the `removeIcon` in addition to `onRemove`. That does seem a little redundant though since the 99% use-case would be to have the `removeIcon` be `delete`. I'll add it back and add the test. I'll just use:

```js
const removeIconShorthand = _.isUndefined(removeIcon) ? 'delete' : removeIcon